### PR TITLE
Fluent website fixes to cross platform text page 

### PR DESCRIPTION
**Pull request by ejlayne**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Fluent website fixes to cross platform text page. Specifically updated copy of usage section, and added additional property tables. Also fixed more broken "related pages" links on cross platform pages.

#### Focus areas to test

Cross platform control pages of Fluent website, specifically Text.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/13131)

### CODE REVIEW

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 1]
```diff
@@ -54,37 +54,82 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
-                        name: 'fontFamily',
+                        name: 'backgroundColor',
                         typeTokens: [
                           {
                             text: 'string',
                           },
                         ],
                         kind: 'property' as 'property',
-                        description: 'Font family of the Text.\n',
+                        description: 'Foreground color of Text.\n',
```
Comment (ksiler) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:108]: Should be background?

Comment (ejlayne) [reply apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx]: Good catch, resolved in new commit.

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 2]
```diff
@@ -95,19 +142,360 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
+                    members: [
+                      {
+                        name: 'captionStandard',
+                        typeTokens: [
+                          {
+                            text: 'VariantValue',
+                          },
+                        ],
+                        kind: 'property' as 'property',
+                        description: 'Used for caption text.',
```
Comment (khmakoto) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:176]:
```suggestion
                        description: 'Used for caption text.\n',
```

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 3]
```diff
@@ -95,19 +142,360 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
+                      },
+                      {
+                        name: 'subheaderStandard',
+                        typeTokens: [
+                          {
+                            text: 'VariantValue',
+                          },
+                        ],
+                        kind: 'property' as 'property',
+                        description: 'Used for subheading text.\n',
```
Comment (khmakoto) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:231]:
```suggestion
                        description: 'Used for subheader text.\n',
```

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 4]
```diff
@@ -95,19 +142,360 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
+                    members: [
+                      {
+                        name: 'caption',
+                        typeTokens: [
+                          {
+                            text: 'FontSizeValuePoints',
+                          },
+                        ],
+                        kind: 'property' as 'property',
+                        description: 'Used for caption text.',
```
Comment (khmakoto) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:327]:
```suggestion
                        description: 'Used for caption text.\n',
```

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 5]
```diff
@@ -95,19 +142,360 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
+                      },
+                      {
+                        name: 'subheader',
+                        typeTokens: [
+                          {
+                            text: 'FontSizeValuePoints',
+                          },
+                        ],
+                        kind: 'property' as 'property',
+                        description: 'Used for subheading text.\n',
```
Comment (khmakoto) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:360]:
```suggestion
                        description: 'Used for subheader text.\n',
```

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 6]
```diff
@@ -95,19 +142,360 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
+                    members: [
+                      {
+                        name: 'regular',
+                        typeTokens: [
+                          {
+                            text: 'FontWeightValue',
+                          },
+                        ],
+                        kind: 'property' as 'property',
+                        description: 'Used for regular text.',
```
Comment (khmakoto) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:412]:
```suggestion
                        description: 'Used for regular text.\n',
```

File: apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx [hunk 7]
```diff
@@ -95,19 +142,360 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
...
+                    members: [
+                      {
+                        name: 'primary',
+                        typeTokens: [
+                          {
+                            text: 'FontFamilyValue',
+                          },
+                        ],
+                        kind: 'property' as 'property',
+                        description: 'Used for primary text in a theme.',
```
Comment (khmakoto) [inline apps/fabric-website/src/pages/Controls/TextPage/TextPage.tsx:443]:
```suggestion
                        description: 'Used for primary text in a theme.\n',
```

**Comment by ejlayne** [comment]
@micahgodbolt @ecraig12345 @khmakoto  can you help me decipher what these tests are and why they're failing? Or, if it's ok to check in with this test failing? Link to test in azure pipeline: 

https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=74273&view=logs&jobId=1387381c-70b4-581c-de41-e64c81c7b28d&j=1387381c-70b4-581c-de41-e64c81c7b28d&t=34ff2578-9ab2-5b52-602b-30614dd518e8

**Comment by khmakoto** [comment]
@ejlayne I don't think it's your PR, these have been failing consistently across multiple PRs. Let me trigger a re-run to see if they pass.

**Comment by micahgodbolt** [comment]
@khmakoto thanks!

**Comment by micahgodbolt** [comment]
yeah, def not something we did

**Comment by micahgodbolt** [comment]
Going to merge into content branch. No need to wait for tests to pass on this PR into a non master branch

**Pull request merged**